token/cli/src/main.rs

```diff
@@ -3624,6 +3652,44 @@ fn app<'a, 'b>(
                 .nonce_args(true)
                 .offline_args(),
         )
+        .subcommand(
+            SubCommand::with_name(CommandName::UpdateMetadataPointerAddress.into())
+                .about("Updates default account state for the mint. Requires the default account state extension.")
+                .arg(
+                    Arg::with_name("token")
+                        .validator(is_valid_pubkey)
+                        .value_name("TOKEN_MINT_ADDRESS")
+                        .takes_value(true)
+                        .index(1)
+                        .required(true)
+                        .help("The address of the token mint to update default account state"),
+                )
+                .arg(
+                    Arg::with_name("metadata_address")
+                        .long("metadata-address")
+                        .value_name("ADDRESS")
+                        .takes_value(true)
+                        .help(
+                            "Specify address that stores token metadata."
+                        ),
+                )
```

Comment:
This will introduce a footgun. As it stands, if someone just accidentally forgets to put in a metadata address, the command will clear it.

Instead, this should look like the `new_authority` flag in `Authorize`, where someone can specify `--disable` to clear the metadata pointer address. Then if `--disable` is *not* provided, the `metadata_address` argument is required.

Reply:
done!